Template10 (Validation)/Validation/ValidatableViewModelBase.cs

```diff
@@ -9,16 +10,17 @@
 
 namespace Template10.Validation
 {
-    public abstract class ModelBase : BindableBase, IModel
+    public abstract class ValidatableViewModelBase : ViewModelBase, IModel
     {
-        public ModelBase()
+        public ValidatableViewModelBase()
```

Comment:
I'm sorry I don't quite understand what do you mean. Your base class name in T10 is ViewModelBase, it's the actual information: https://github.com/Windows-XAML/Template10/blob/master/Template10%20(Library)/Mvvm/ViewModelBase.cs . Do you plan to change the name of the base class ViewModelBase to ModelBase in the core Template10 Library?